pypeit/pypeit.py

```diff
@@ -864,38 +861,37 @@ def extract_one(self, frames, det, sciImg, objFind, initial_sky, sobjs_obj):
         else:
             final_global_sky = objFind.global_skysub(previous_sky=initial_sky, skymask=skymask, show=self.show)
 
+
         msgs.info("Extraction begins for {} on det={}".format(self.basename, det))
 
         # Instantiate Reduce object
         # Required for pypeline specific object
         # At instantiaton, the fullmask in self.sciImg is modified
         # TODO Are we repeating steps in the init for FindObjects and Extract??
-        self.redux = extraction.Extract.get_instance(
+        self.exTract = extraction.Extract.get_instance(
             sciImg, sobjs_obj, self.spectrograph, 
             self.par, self.caliBrate, self.objtype, 
             bkg_redux=self.bkg_redux, 
-            find_negative=self.find_negative,
             std_redux=self.std_redux,
```

Comment:
Did you mean not to include `return_negative` here? Is it because it's always `False`?

Reply:
Yeah in pypeit we just want the default behavior, which is return_negative == False so I did not include it in the call here.